src/main/java/org/agbrothers/englishbot/constant/CommonPhrase.java

```diff
@@ -4,9 +4,28 @@
 public final class CommonPhrase {
     private CommonPhrase() {}
 
+    public static final String MAIN_MENU = "/main_menu";
     public static final String ERROR_MESSAGE = "Сталась прикра технічна помилка. " +
-            "Спробуйте перейти в головне меню - " + LinkLabel.MAIN_MENU;
-    public static final String RETURN_MAIN_MENU = "  Для повернення в головне меню натисніть " + LinkLabel.MAIN_MENU;
-    public static final String TYPED_WORD_ERROR = "Помилка в слові англійською або перекладі слова. +" +
-            "Слово англійською має бути латиницею, переклад - лише кирилицею.";
+            "Спробуйте перейти в головне меню - " + MAIN_MENU;
+    public static final String RETURN_MAIN_MENU = "  Для повернення в головне меню натисніть " + MAIN_MENU;
+    public static final String TYPED_WORD_ERROR = "Помилка в слові англійською або перекладі слова. Слово англійською має бути латиницею, переклад - лише кирилицею.";
+    public static final String LESSON_ENDING = "Урок закінчено. Правильних відповідей - %d із %d.\n"; //count of words in lesson
+    public static final String CORRECT_ANSWER = "Ви відповіли правильно! \n";
+    public static final String INCORRECT_ANSWER = "Відповідь не вірна! Правильна відповідь - ";
+    public static final String MAKE_CHOICE = "Будь ласка, зробіть свій вибір";
+    public static final String HELLO_MESSAGE = "Привіт, я дуже радий, що ти хочеш покращити свої знання англійської мови!!!" +
+            " Маю надію, що я тобі в цьому допоможу!)) " +
+            "Для початку навчання натисни " + MAIN_MENU;
+    public static final String YOU_CHOOSE = "Ви вибрали ";
+    public static final String INCORRECT_MESSAGE = "Не розумію, що ти маєш на увазі? ";
+    public static final String DEVELOPMENT_OF_DIRECTION = " Триває розробка даного напрямку ";
```

Comment:
Rename this to something more meaningful, i.e. DEVELOPMENT_IN_PROGRESS

Reply:
Thanks. Done